squad-server/plugins/discord-interval-updated-message.js

```diff
@@ -0,0 +1,111 @@
+import DataTypes from 'sequelize';
+import DiscordBasePlugin from './discord-base-plugin.js';
+
+export default class DiscordIntervalUpdatedMessage extends DiscordBasePlugin {
+  static get optionsSpecification() {
+    return {
+      ...super.optionsSpecification,
+      subscribeMessage: {
+        description: 'Trigger message to start the broadcast on the channel',
+        default: '!start'
+      },
+      unsubscribeMessage: {
+        description: 'Trigger message to stop the broadcast on this channel',
+        default: '!stop'
+      },
+      interval: {
+        description: 'Update interval in seconds',
+        default: 300
+      },
+      storage: {
+        description: 'Storrage connector name',
+        connector: 'databaseClient',
+        default: 'sqlite'
+      }
+    };
+  }
+
+  constructor(server, options) {
+    super(server, options);
+
+    this.subscribeMessage = options.subscribeMessage;
+    this.unsubscribeMessage = options.unsubscribeMessage;
+    this.interval = options.interval;
+    this.storage = options.storage;
+
+    this.discordClient.once('ready', async () => {
+      await this.setupDatabase();
+      this.setupMessageTriggers(server);
+      this.setupUpdateInterval(server);
+    });
+  }
+
+  async setupDatabase() {
+    try {
+      const tableName = this.constructor.name + '_DiscordIntervalUpdateMessage';
+      this.storage.authenticate();
```

Comment:
Would this be better placed where the sequelize instance is originally created? Otherwise, the auth call will be duped across multiple plugins?